src/zenml/models/model_base_model.py

```diff
@@ -33,31 +33,39 @@ class ModelBaseModel(BaseModel):
     license: Optional[str] = Field(
         title="The license model created under",
         max_length=TEXT_FIELD_MAX_LENGTH,
+        default=None,
```

Comment:
Same question as above - None is the default for Optional.

Reply:
Same answer as above ;)